[doc] AOTI debugging guide

henrylhtsang opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #160430

Folded from https://discuss.pytorch.org/t/a-beginners-guide-to-debugging-aot-inductor-cuda-illegal-memory-access/222188

cc @svekars @sekyondaMeta @AlannaBurke

henrylhtsang committed:
[doc] AOTI debugging guide

[ghstack-poisoned]

AlannaBurke commented on docs/source/_static/img/aoti_debugging_guide/cuda_ima_cca.png:
yes please! you don't need to use a caption tag, you can just add the text below the image or something. just trying to keep things accessible!

henrylhtsang replied:
@AlannaBurke can you share an example of caption so I can copy

AlannaBurke replied:
Sorry I didn't see this sooner - there's an example here that should do it: https://myst-parser.readthedocs.io/en/latest/syntax/roles-and-directives.html#how-directives-parse-content

henrylhtsang committed:
Update on "[doc] AOTI debugging guide"


Folded from https://discuss.pytorch.org/t/a-beginners-guide-to-debugging-aot-inductor-cuda-illegal-memory-access/222188

cc svekars sekyondaMeta AlannaBurke

[ghstack-poisoned]

henrylhtsang commented:
@pytorchbot merge
